committeeoversightapp/views.py

```diff
@@ -77,12 +77,6 @@ class EventListJson(BaseDatatableView):
     # define the columns that will be returned
     columns = ['updated_at', 'name', 'start_date', 'id', 'id']
 
-    # define column names that will be used in sorting
```

Comment:
why is this removed?

Reply:
Defining which columns are sortable is something that theoretically can be done in either the `django-datatables-view` package or in DataTables itself through the [orderable](https://datatables.net/reference/option/columns.orderable) parameter. I had tried doing it with the package with this line, taken straight from the documentation, but it didn't work! I switched to the DataTables method and it did.